pkg/core/storage/memcached_store.go

```diff
@@ -15,13 +15,25 @@ import (
 type MemCachedStore struct {
 	MemoryStore
 
+	nativeCacheLock sync.RWMutex
+	nativeCache     map[int32]NativeContractCache
```

Comment:
Why do we still have it in the `MemCachedStore`?

Reply:
Fixed in a separate commit.